dags/infra/sns_topics.py

```diff
@@ -0,0 +1,14 @@
+"""
+# Variables provided from infra to dags
+"""
+from airflow.models import Variable
+
+LANDSAT_SYNC_SNS_TOPIC_ARN = Variable.get(
+    "landsat_sns_topic",
+    "arn:aws:sqs:af-south-1:565417506782:deafrica-prod-af-eks-index-landsat-scene",
+)
+
+SYNC_SENTINEL_2_CONNECTION_TOPIC_ARN = Variable.get(
```

Comment:
can we start all sentinel_2 related variables with `SENTINEL_2...`

Reply:
Absolutely yes. Naming convention should always come from the cloud team, please send me the names that you want for buckets, topics, and queues and I'll change them.
Thank you

Reply:
Let's start following variable naming convention if not so we can manage it better, like: 
`SENTINEL_2_...`
`LANDSAT_...`